src/mono/wasm/templates/templates/browser/Program.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Runtime.CompilerServices;
+
+Console.WriteLine ("Hello, Console!");
```

Comment:
does that top level statement work?

Reply:
Yes. It's just compile-time feature and it works on wasm runtime.

Reply:
It works fine. I used it to make the code simple and closer to the desktop console sample.